policy/modules/system/systemd.te

```diff
@@ -1848,6 +1848,10 @@ systemd_log_parse_environment(systemd_sysusers_t)
 
 systemd_stream_connect_nsresourced(systemd_sysusers_t)
 
+# package upgrade
+userdom_use_all_users_fds(systemd_sysusers_t)
+userdom_use_inherited_user_terminals(systemd_sysusers_t)
```

Comment:
Can you clarify what is happening here exactly? I'd think errors would go to the journal.

Reply:
It think during package upgrade via apt, dpkg inherits the terminal, and leaves it for package post installation scripts, e.g. systemd-sysusers:

```
type=PROCTITLE msg=audit(03/11/24 12:04:07.787:321) : proctitle=systemd-sysusers systemd-coredump.conf 
type=PATH msg=audit(03/11/24 12:04:07.787:321) : item=1 name=/lib64/ld-linux-x86-64.so.2 inode=401892 dev=fe:01 mode=file,755 ouid=root ogid=root rdev=00:00 obj=system_u:object_r:ld_so_t:s0 nametype=NORMAL cap_fp=none cap_fi=none cap_fe=0 cap_fver=0 cap_frootid=0 
type=PATH msg=audit(03/11/24 12:04:07.787:321) : item=0 name=/usr/bin/systemd-sysusers inode=400196 dev=fe:01 mode=file,755 ouid=root ogid=root rdev=00:00 obj=system_u:object_r:systemd_sysusers_exec_t:s0 nametype=NORMAL cap_fp=none cap_fi=none cap_fe=0 cap_fver=0 cap_frootid=0 
type=CWD msg=audit(03/11/24 12:04:07.787:321) : cwd=/ 
type=EXECVE msg=audit(03/11/24 12:04:07.787:321) : argc=2 a0=systemd-sysusers a1=systemd-coredump.conf 
type=SYSCALL msg=audit(03/11/24 12:04:07.787:321) : arch=x86_64 syscall=execve success=yes exit=0 a0=0x55af762d3130 a1=0x55af762d2e48 a2=0x55af762d2fe8 a3=0x8 items=2 ppid=4388 pid=4421 auid=root uid=root gid=root euid=root suid=root fsuid=root egid=root sgid=root fsgid=root tty=(none) ses=1 comm=systemd-sysuser ex
e=/usr/bin/systemd-sysusers subj=unconfined_u:unconfined_r:systemd_sysusers_t:s0-s0:c0.c1023 key=(null) 
type=AVC msg=audit(03/11/24 12:04:07.787:321) : avc:  denied  { use } for  pid=4421 comm=systemd-sysuser path=/dev/pts/4 dev="devpts" ino=7 scontext=unconfined_u:unconfined_r:systemd_sysusers_t:s0-s0:c0.c1023 tcontext=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 tclass=fd permissive=0 
type=AVC msg=audit(03/11/24 12:04:07.787:321) : avc:  denied  { use } for  pid=4421 comm=systemd-sysuser path=/dev/pts/4 dev="devpts" ino=7 scontext=unconfined_u:unconfined_r:systemd_sysusers_t:s0-s0:c0.c1023 tcontext=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 tclass=fd permissive=0 
type=AVC msg=audit(03/11/24 12:04:07.787:321) : avc:  denied  { use } for  pid=4421 comm=systemd-sysuser path=/dev/pts/4 dev="devpts" ino=7 scontext=unconfined_u:unconfined_r:systemd_sysusers_t:s0-s0:c0.c1023 tcontext=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 tclass=fd permissive=0 
type=AVC msg=audit(03/11/24 12:04:07.787:321) : avc:  denied  { read write } for  pid=4421 comm=systemd-sysuser path=/dev/pts/4 dev="devpts" ino=7 scontext=unconfined_u:unconfined_r:systemd_sysusers_t:s0-s0:c0.c1023 tcontext=unconfined_u:object_r:user_devpts_t:s0 tclass=chr_file permissive=0
```